bittensor/_dendrite/dendrite_impl.py

```diff
@@ -643,14 +668,20 @@ def to_wandb(self):
         """
         wandb_info = {}
 
+        # ---- Dendrite response histograms for return_ops and query_times
+        response_stats = [[u, t, r, codes.code_to_string(r)] for u, t, r in
+                          zip(self.stats.uids, self.stats.query_times, self.stats.return_ops)]
+        wandb_table = wandb.Table(data=response_stats, columns=['uids', 'query_times', 'return_ops', 'return_ops_str'])
+        wandb_info['query_times'] = wandb.plot.histogram(wandb_table, 'query_times', 'Dendrite query response times')
+
         # ---- Dendrite stats per pubkey for wandb 
         for uid in self.stats.requested_peers_count.keys():
             respond_rate = self.stats.responded_peers_count[uid].value / self.stats.requested_peers_count[uid].value
-           
+
             uid_str = str(uid.item()).zfill(3)
-            wandb_info[f'dend_quested uid: {uid_str}']= self.stats.requested_peers_count[uid].value
-            wandb_info[f'dend_responded uid: {uid_str}']= self.stats.responded_peers_count[uid].value
-            wandb_info[f'dend_respond_time uid: {uid_str}']= self.stats.peers_respond_time[uid].value
-            wandb_info[f'dend_respond_rate uid: {uid_str}']= respond_rate
+            wandb_info[f'dend_requested uid: {uid_str}'] = self.stats.requested_peers_count[uid].value
```

Comment:
It would be great if we could figure out how to join these onto a single graph also. As is, we create too many individual graphs.